requirements/base.in

```diff
@@ -19,6 +19,7 @@ event-tracking
 mako
 openedx-events                      # Open edX Events from Hooks Extension Framework (OEP-50)
 openedx-filters
+pydantic==2.7.1
```

Comment:
why are you pinning the version here ?

Reply:
https://github.com/eduNEXT/eox-nelp/blob/8a07da60642c7d46a67ad80f9aa413a7400c679f/setup.py#L91

Reply:
Let ask again, why are you pinning the last available version ?  I didn't have to install that in my local machine, did you have to install that ?

Reply:
Yes, I think also image doesnt have it.
![image](https://github.com/eduNEXT/eox-nelp/assets/51926076/e639ee60-bf62-4188-b46a-e8dc3bca4f83)

Reply:
What do you think of removing that and set  the version into the strain file ?

Reply:
but the test would not work for the requirements?

Reply:
I refer to the pinned version 
```suggestion
pydantic
```